docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
+  will not send logs directly to Elasticsearch, instead it will use
+  `secure_forward` to send logs to mux.
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
+  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
+  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
```

Comment:
The original idea for mux was to be a normalizer/enricher component.  I would like to get back to that if/when we solve the performance problems.
Another concern that has been raised by members of the OpenShift team is that they want the log agent running on the node to be as lightweight as possible - no processing - just ship the logs off of the node as fast as possible.  With `full_no_k8s_meta` as opposed to `minimal` or `USE_MUX_CLIENT=true` we're going in the wrong direction.